geckomat/utilities/integrate_proteomics/generate_protModels.m

```diff
@@ -88,9 +85,10 @@ function generate_protModels(ecModel,grouping,name,ecModel_batch)
     cd ../kcat_sensitivity_analysis
     ecModelP  = changeMedia_batch(ecModel,c_source);
     tempModel = changeMedia_batch(ecModel_batch,c_source);
-    cd ../limit_proteins
-    %Calculate f-factor, after having filtered data. 
-    f = measureAbundance(ecModel.enzymes,pIDs,abundances);
+    cd ../limit_proteins  
+    if isempty(f) | f==0
```

Comment:
Even if `f = []` in line 52, when `i = 2`, `f` is no longer empty. Not sure how to fix, resetting `f` at the end of the `i`-loop would also overwrite any `f` that could have been set in line 48.